nautobot/extras/models/relationships.py

```diff
@@ -329,16 +330,54 @@ def required_related_objects_errors(
 class RelationshipManager(BaseManager.from_queryset(RestrictedQuerySet)):
     use_in_migrations = True
 
-    def get_for_model(self, model):
+    @lru_cache(maxsize=128)
+    def get_for_model(self, model, hidden=False):
         """
         Return all Relationships assigned to the given model.
+
+        Args:
+            model: The django model to which relationships are registered
+            hidden: Boolean to look for hidden relationships
```

Comment:
Feels weird to me that `hidden` is always either `True` or `False` here, so in either case you only get a subset of all relationships -- shouldn't there be a way to get all relationships regardless of their `hidden` flag?